### PR TITLE
Generate javacript and css in the debian packaging

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Build javascript and css in debian packaging

# Impacts
Javascript / CSS

# Delete branch after merge
YES

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -169,9 +169,35 @@ install: build
...
 	done
+	#Portal javascript/css
+	make -C html/common/ vendor
+	make -C html/common light-dist
+	install -d -m0755 $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common
+	install -m0644 $(CURDIR)/html/common/styles.css $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	install -m0644 $(CURDIR)/html/common/styles.css.map $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	# Admin javascript/css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css.map
```
Comment (cgx) [inline debian/rules]: You should move the above two lines in the *Portal* section, just to make it clear.

File: debian/rules [hunk 2]
```diff
@@ -169,9 +169,35 @@ install: build
 	for i in `find * -path 'docs/images/*' -type f`; do \
 		install -m0644 $$i $(CURDIR)/debian/packetfence-doc$(PREFIX)/$(NAME)/html/pfappserver/root/static/images/; \
 	done
+	#Portal javascript/css
+	make -C html/common/ vendor
+	make -C html/common light-dist
```
Comment (cgx) [inline debian/rules:176]: We also need grunt (`sudo npm install -g grunt-cli`). Should we install it from the Makefile?

File: debian/rules [hunk 3]
```diff
@@ -169,9 +169,35 @@ install: build
...
+	install -d -m0755 $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common
+	install -m0644 $(CURDIR)/html/common/styles.css $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	install -m0644 $(CURDIR)/html/common/styles.css.map $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	# Admin javascript/css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css.map
+	make -C html/pfappserver/root/static/ vendor
```
Comment (cgx) [inline debian/rules:181]: We also need grunt (`sudo npm install -g grunt-cli`) and bower (`sudo npm install -g bower`). Should we install them from the Makefile?

File: debian/rules (hunk 2 above)
Comment (fdurand) [reply debian/rules:176]: I did it in the chroot directly but i also changed in the Makefile

File: debian/rules (hunk 3 above)
Comment (fdurand) [reply debian/rules:181]: I did it in the chroot directly but i also changed in the Makefile

**Pull request merged**